puma/tests/hlplots/test_results.py

```diff
@@ -466,17 +464,8 @@ def test_plot_fraction_scans_bjets_eff(self):
             results = Results(signal="bjets", sample="test", output_dir=tmp_file)
             results.add(self.dummy_tagger_1)
             results.plot_fraction_scans(rej=False, optimal_fc=True)
-            self.assertEqual(
-                None,
-                compare_images(
-                    os.path.join(
-                        os.path.dirname(__file__),
-                        "..",
-                        "expected_plots/test_bjets_fraction_scan.png",
-                    ),
-                    results.get_filename("fraction_scan"),
-                    tol=1,
-                ),
+            self.assertIsFile(
+                Path(tmp_file) / "test_bjets_fraction_scan_back_cjets_ujets_eff_70p0_scan_fc.png"
```

Comment:
can we clean up the filenames a bit?
- `test_bjets_fraction_scan_back_cjets_ujets_eff_70p0_scan_fc` could probably just be
- `test_bjets_fcscan_cjets_ujets_eff70`

also for the profile
- `test_cjets_cjets_eff_vs_pt_profile_flat_bjets_10_rej_per_bin` could be 
- `test_cjets_cjets_eff_vs_pt_bjets_rej_flat_10`

and similarly
- `test_cjets_cjets_eff_vs_pt_wp_0p7_profile_fixed_cut` could be 
- `test_cjets_cjets_eff_vs_pt_fixed_cut_wp70` or something